src/Microsoft.AspNet.SignalR.SqlServer/SqlServerServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.SignalR;
+using Microsoft.AspNet.SignalR.Messaging;
+using Microsoft.AspNet.SignalR.SqlServer;
+using Microsoft.Framework.ConfigurationModel;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    public static class SqlServerServiceCollectionExtensions
+    {
+        public static IServiceCollection AddSignalRSqlServer(this IServiceCollection services, Action<SqlScaleoutConfiguration> configureOptions = null)
```

Comment:
Before I thought to change it in https://github.com/aspnet/SignalR-SqlServer/pull/2 after  https://github.com/aspnet/SignalR-Server/pull/65, now rename it.